analyze.js

```diff
@@ -33,15 +33,13 @@ function analyze(options, callback){
 
   command = buildCommand(options);
 
-  var outPutFileName = (options.url) ? options.url.replace(/^(https?|ftp):\/\//, '') + '.json' : 'results.json';
-
-  exec(command + ' --reporter=json > ' + outPutFileName, function(error, stdout, stderr){
+  exec(command + ' --reporter=json > ' + options.outputFileName, function(error, stdout, stderr){
```

Comment:
I moved that line over to line 27 in file louis.js. Isnt it working for you anymore?